internal/wrappers/client.go

```diff
@@ -579,6 +579,21 @@ func request(client *http.Client, req *http.Request, responseBody bool) (*http.R
 			logger.PrintIfVerbose(err.Error())
 		}
 		if resp != nil && err == nil {
```

Comment:
Suggestion:
Consider checking err!=nil and return the error before checking resp != nil

Reply:
There is a check on this one line above